apps/core/helpers.py

```diff
@@ -17,9 +17,9 @@ def send_email(user: User, subject, message):
     email.send()
 
 
-def send_verification_email(user, url: str):
+def send_verification_email(user, url: str, email=""):
```

Comment:
I can suggest making 'email' a required argument in this function, this way function will behave similar in all calls and it would become more explicit. It will require updating calls to it, but we have only 4 of them